tools/generate-pdfs.sh

```diff
@@ -16,31 +16,34 @@ set -ex
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function generate_pdfs_from_md() {
+		inputMdFile=$1
+		outputPdfFile=$2
+		configYamlFile=$3
+		headingLineNum=$(awk '/<!---END_OF_HTML_HEADER--->/ { print NR; exit }' $inputMdFile)
+		
+		tail -n +$headingLineNum $inputMdFile | pandoc --template=tools/acle_template.tex --metadata-file=$configYamlFile -o $outputPdfFile
+}
+
 mkdir -p pdfs
 
-#rst2pdf morello/morello.rst         \
+# the option`--inline-footnotes` is used to print the footnotes off
+# the references "in place" in the `References` section.
+#rst2pdf main/acle.rst         \
+#	--inline-footnotes \
 #	-s tools/rst2pdf-acle.style \
 #        --repeat-table-rows         \
 #        --default-dpi=110           \
-#        -o pdfs/morello.pdf
+#        -o pdfs/acle.pdf
 
-# the option`--inline-footnotes` is used to print the footnotes off
-# the references "in place" in the `References` section.
-rst2pdf main/acle.rst         \
-	--inline-footnotes \
-	-s tools/rst2pdf-acle.style \
-        --repeat-table-rows         \
-        --default-dpi=110           \
-        -o pdfs/acle.pdf
-
-rst2pdf neon_intrinsics/advsimd.rst         \
-	-s tools/rst2pdf-acle-intrinsics.style \
-        --repeat-table-rows         \
-        --default-dpi=110           \
-        -o pdfs/advsimd.pdf
-
-#rst2pdf mve_intrinsics/mve.rst         \
+#rst2pdf neon_intrinsics/advsimd.rst         \
```

Comment:
Why is this part commented out? Don't we still want to generate acle.pdf and advsimd.pdf?